src/pages/base/markets/components/RateHistoryChart.js

```diff
@@ -9,7 +9,7 @@ import { compact } from "../../../../utils/number.js";
 function RateHistoryChart(props) {
   const { symbol, timePeriod } = props;
   const { data, isLoading, isError, ErrorFallbackComponent } = useFetch(
-    `markets/${symbol}/historic-details/`,
+    `market/${symbol}/historic-details/`,
```

Comment:
Keep the old views as they were. You need to create new ones for new versions

Reply:
yes this should be markets/